sdk/spring/azure-spring-cloud-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/servicebus/AzureServiceBusAutoConfigurationTest.java

```diff
@@ -97,6 +98,23 @@ public void testTransportTypeWithAmqpWebSockets() {
             });
     }
 
+    @Test
```

Comment:
Can we add UTs in the queue&topic autoconfigurtion to see if the client factory beans are created as the way the properties are configured?